.github/workflows/test_destination_clickhouse.yml

```diff
@@ -67,12 +66,49 @@ jobs:
       - name: create secrets.toml
         run: pwd && echo "$DLT_SECRETS_TOML" > tests/.dlt/secrets.toml
 
+      # OSS ClickHouse
+      - run: |
+          docker-compose -f ".github/clickhouse-compose.yml" up -d
+          echo "Waiting for ClickHouse to be healthy..."
+          timeout 30s bash -c 'until docker-compose -f ".github/clickhouse-compose.yml" ps | grep -q "healthy"; do sleep 1; done'
+          echo "ClickHouse is up and running"
+        name: Start ClickHouse OSS
+
+
+      - run: poetry run pytest tests/load -m "essential"
```

Comment:
This is missing `if` conditions. Look at https://github.com/dlt-hub/dlt/blob/af7752725b34ff9fa6f081f1c2483ac767944284/.github/workflows/test_destinations.yml#L86.

The essential tests run twice without such conditions.

Reply:
Thank you Jorrit! I missed this one.